### ISSUE TITLE
sync between repositorys with different backend

### ISSUE THREAD

**Issue by kisst** [restic/restic#2725]
Output of `restic version`
--------------------------
$ restic version
restic 0.9.6 compiled with go1.13.4 on linux/amd64

How did you run restic exactly?
-------------------------------
One repository on S3, synced to local filesystem ( NFS mount ) 
But commands might tell the story better.
```
$ restic -r s3:s3.amazonaws.com/my-own-bucket-name unlock
enter password for repository: 
repository a******z opened successfully, password is correct
successfully removed locks
$ aws s3 sync s3://my-own-bucket-name /backup_target/
$ restic -r /backup_target unlock
enter password for repository: 
Fatal: config cannot be loaded: ciphertext verification failed. Try again
enter password for repository: 
Fatal: config cannot be loaded: ciphertext verification failed. Try again
enter password for repository: 
Fatal: Fatal: config cannot be loaded: ciphertext verification failed
```
I used the unlock function, as that's what I found the lead IO heavy, in in case of S3 also the cheapest and quickest, but all other `ls key` `snapshoots` etc all fail with the same error.

What backend/server/service did you use to store the repository?
----------------------------------------------------------------

AWS S3  + local file system ( NFS mount point ) 

Expected behavior
-----------------
Sync between different type of repos should still work.
Use-case, quick local backup to NFS then the NAS sync up to S3 later for safe keeping, but both should be usable, just in case.

Actual behavior
---------------
Currently it does not work after sync

Steps to reproduce the behavior
-------------------------------
 - Create a repo on S3
 - Sync down the content of the bucket
 - Try to use the local copy.

Do you have any idea what may have caused this?
-----------------------------------------------

Not really

Do you have an idea how to solve the issue?
-------------------------------------------

Neither

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------
Oh yeah, It's always good to see the backup ticking, it's reassuring to know that I have a way back.

**Comment by MichaelEischer** [comment]
That looks like the `config` file in `/backup_target/` is damaged. Could you compare the config file with the one stored in S3? Which files and directory do you have in the root folder of your local backup copy (`ls -la /backup_target/`)?

**Comment by kisst** [comment]
Thanks that was in deed the src of the error.

I downloaded again the config file alone from S3 and run a `diff local-config s3-config` and I got:
```
Binary files local-config and s3-config differ 
```
 so I overwrote the local, and now the repo works fine.

So the error is somewhere around the S3 sync, I reviewed the[ docs](https://docs.aws.amazon.com/cli/latest/reference/s3/sync.html) to see if there is any flag which would have possible spot that the files are different, but I can't see it, unsure how the binary got corrupted, but for sure the `aws s3 sync` did not pick it up, that they are different.

So the only thing what on restic side we might able to do is have a more descriptive error message, or add it to the FAQ, but I guess this GH issue will be also a good reference point.

I will try to reproduce it, and raise a separated ticket with AWS .
